src/Microsoft.AspNet.Mvc.Core/Infrastructure/MvcRouteHandler.cs

```diff
@@ -87,7 +87,7 @@ public VirtualPathData GetVirtualPath(VirtualPathContext context)
                 {
                     _logger.ExecutingAction(actionDescriptor);
 
-                    var startTime = Environment.TickCount;
+                    var startTime = DateTime.UtcNow;
```

Comment:
Curious: what's the performance impact of this change?

`DateTime.UtcNow` is known to be rather slow, which might not be appropriate in such a critical code path.
Maybe it would be better to only call `DateTime.UtcNow` after a quick `logger.IsEnabled` check?